src/main/java/net/mangolise/parkourinfinite/ParkourInfGame.java

```diff
@@ -24,13 +25,10 @@ public ParkourInfGame(ParkourInfGame.Config config) {
     public void setup() {
         super.setup();
 
-        DimensionType dimension = DimensionType.builder().ambientLight(15).build();
-        DynamicRegistry.Key<DimensionType> dim = MinecraftServer.getDimensionTypeRegistry().register("parkour-infinite", dimension);
-
         // Player spawning
         GlobalEventHandler events = MinecraftServer.getGlobalEventHandler();
         events.addListener(AsyncPlayerConfigurationEvent.class, e -> {
-            Instance instance = MinecraftServer.getInstanceManager().createInstanceContainer(dim, null);
```

Comment:
We're losing the fullbright dimension type - this needs to either be retained or justified.